### PR TITLE
EVG-6572: separate module commit messages

### PR DESCRIPTION
**Pull request by ybrill**
Add a new option to the commit queue's set-module subcommand to supply a message for the commit that will be created in the module's repo. If a message is not supplied the message of the existing commit is used.

This also makes it easier to decouple the main patch's commit message from the patch description down the line. For instance, if down the line we decide to take just the commit message's subject as the patch description in the case of multi-line messages.

### CODE REVIEW

File: service/api_patch.go [hunk 1]
```diff
@@ -147,21 +147,22 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var moduleName, patchContent, githash string
+	var moduleName, patchContent, githash, message string
 
 	if r.Header.Get("Content-Type") == formMimeType {
-		moduleName, patchContent, githash = r.FormValue("module"), r.FormValue("patch"), r.FormValue("githash")
+		moduleName, patchContent, githash, message = r.FormValue("module"), r.FormValue("patch"), r.FormValue("githash"), r.FormValue("message")
```
Comment (tychoish) [inline service/api_patch.go]: nit: can you put this on multiple lines?

File: operations/http.go [hunk 2]
```diff
@@ -293,12 +293,13 @@ func (ac *legacyClient) DeletePatchModule(patchId, module string) error {
 }
 
 // UpdatePatchModule makes a request to the API server to set a module patch on the given patch ID.
-func (ac *legacyClient) UpdatePatchModule(patchId, module, patch, base string) error {
+func (ac *legacyClient) UpdatePatchModule(patchId, module, patch, base, message string) error {
```
Comment (tychoish) [inline operations/http.go]: it wouldn't be a bad thing if there was a struct that you passed to this to get ~keyed args.

File: service/api_patch.go (hunk 1 above)
Comment (ybrill) [reply service/api_patch.go]: done.

File: operations/http.go (hunk 2 above)
Comment (ybrill) [reply operations/http.go]: done.

**Pull request merged**